modules/apps/frontend-theme/kyoung-liferay-theme/src/css/components/_navbars.scss

```diff
@@ -0,0 +1,61 @@
+.navbar-top {
+	padding-bottom: 0.907rem;
+	padding-top: 0.907rem;
+}
+
+.navbar-blank {
+	flex-wrap: wrap;
+}
+
+.navbar-classic {
+	background-color: #FFF;
+
+	&.navbar-light {
+		border-bottom: 1px solid $light-color;
+	}
+
+	.navbar-blank {
+		@include media-query(null, $screen-xs-max) {
+			margin-top: 2rem;
+		}
+	}
```

Comment:
We should write this so that we do not have a _blank_ CSS selector.
Ex.
```
@include media-query(null, $screen-xs-max) {
	.navbar-blank {
		margin-top: 2rem;
	}
}
```